### PR TITLE
PT-163485144 Expose log/events in ContractCallObject in HTTP API

### PR DESCRIPTION
**Pull request by hanssv**

### CODE REVIEW

**Comment by hanssv** [comment]
This should be rebased on `fortuna` once master has been merged to `fortuna` I guess... Don't review just yet...

File: apps/aecontract/src/aect_call.erl [hunk 1]
```diff
@@ -198,9 +198,9 @@ serialize_for_client(#call{caller_id    = CallerId,
      }.
 
 serialize_log_entry_for_client({Address, Topics, Data}) ->
-    #{ <<"address">> => Address
-     , <<"topics">>  => Topics
-     , <<"data">>    => Data
+    #{ <<"address">> => aehttp_api_encoder:encode(contract_pubkey, Address)
+     , <<"topics">>  => [ Topic || <<Topic:256>> <- Topics ]
```
Comment (lucafavatella) [inline apps/aecontract/src/aect_call.erl]: This syntax is filtering out Topics that do not match `<<_:256>>`: who guarantees that this constraint holds for all topics? Or does code here intend to filter?

Comment (lucafavatella) [reply apps/aecontract/src/aect_call.erl]: Could you please track future improvement of restricting module to check that stored log (assert_field?) matches this expectation.

Comment (hanssv) [reply apps/aecontract/src/aect_call.erl]: All topics are of VM type `word`

Comment (lucafavatella) [reply apps/aecontract/src/aect_call.erl]: https://www.pivotaltracker.com/story/show/164431952

**Pull request merged**